contexts/webledger-v1.jsonld

```diff
@@ -115,6 +115,7 @@
     "recommendedElector": {"@id": "wl:recommendedElector", "@container": "@set"},
     "requiredProof": "sec:requiredProof",
     "revoked": {"@id": "sec:revoked", "@type": "xsd:dateTime"},
+    "sequence": {"@id": "wl:sequence", "@type": "xsd:integer"},    
```

Comment:
@msporny had a chat about this, we think we should use the JSON-LD patch vocabulary for this, so we should do: `"sequence": {"@id": "jldp:sequence", "@type": "xsd:integer"}`. Our plan moving forward is to reuse this quite a bit in a number of places (other than just webledger stuff) and just using the one vocab term for it seems best... and having it live in the JSON-LD patch vocabulary seems like the most reasonable place (we don't have any other vocab for it to live in that makes more sense).